CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contribution Guide for dbt-ClickHouse Adapter
+
+## Introduction
+
+Thank you for considering contributing to the dbt-ClickHouse adapter! We value your contributions and appreciate your
+efforts to improve the adapter. This guide will help you get started with the contribution process.
+
+## Getting Started
+
+### 1. Fork the Repository
+
+Start by forking the repository on GitHub. This will create a copy of the repository under your own GitHub account.
+
+### 2. Set Up Environment
+
+* Make sure Python is installed locally.
+* Create a dedicated virtual environment (optional but recommended)
+* Install all the development requirements:
+    ```bash
+    pip install -r dev_requirements.txt
+    ```
+* Install the local project as a package - go to the project root folder and run
+    ```bash
+      pip install .
+    ```
+* Verify the package was installed successfully:
+  ```bash
+    pip list || grep dbt-clickhouse
+    ```
+  the package will be directed to your local project.
+
+### 3. Create a Branch
+
+Create a new branch for your feature or bug fix, please make sure to follow 
+
+```bash
+git checkout -b my-new-feature
+```
+
+### 4. Make Your Changes
+
+Make the necessary changes in your branch. Ensure that your code follows the existing style and conventions used in the
+project.
+
+### 5. Add or Adjust Tests
+
+The project tests are located under the `test` folder. Please look for the relevant test associated with
+your changes, and adjust it. If not such test, please create one.
+
+### 6. Run The Tests
+
+See [Running Tests](#running-tests) for more information.
+
+> **Important:** Please make sure the tests are running successfully before pushing your code.
+
+### 7. Create a PR
+Create a pull request from your forked repository to the main one. In case this is your first contribution, make sure to sign ClickHouse's CLA.
```

Comment:
optional: ask to add a sensible description of a feature and the use case

Reply:
I guess It would be good idea to stick to some commit message format, like in https://www.conventionalcommits.org for example. Like `fix: ...`, `docs: ...`

Reply:
Done